coverage/.resultset.json

```diff
@@ -1,7 +1,52 @@
 {
   "RSpec": {
     "coverage": {
+      "/Users/dominicpadula/turing/Mod3/projects/yardsourcing/yardsourcing-sendgrid/spec/sendgrid_api_spec.rb": {
```

Comment:
It appears we need to add coverage to the git ignore file here?

Reply:
Ya good call on that. We can fix that asap

Reply:
@jordanfbeck0528 we have coverage marked in our .gitignore file, so I am not sure why that showed up.. 🤷🏽‍♀️

Reply:
I double checked that coverage is in the git ignore in my last commit. should be good to go!

Reply:
sounds good